src/components/PluggableComponent/utils.js

```diff
@@ -0,0 +1,10 @@
+export const isPluginAvailable = async (pluginName) => {
+  if (!pluginName) { return false; }
```

Comment:
I think in this case the `{}` is not necessary

```suggestion
  if (!pluginName) return false;
```

Reply:
I had an issue with the linter using it like this
`return false;`